DataFormats/PatCandidates/interface/MET.h

```diff
@@ -200,11 +247,25 @@ namespace pat {
       // ---- holder for pfMET specific info ---
       std::vector<SpecificPFMETData> pfMET_;
 
+      // MET significance
+      double metSig_;
+      
+      const PackedMETUncertainty findMETTotalShift(MET::METCorrectionLevel cor, MET::METUncertainty shift) const;
+   
+      std::map<MET::METCorrectionLevel, std::vector<MET::METCorrectionType> > corMap_;
+      void initCorMap();
+
+    
+
     protected:
 
       // ---- non-public correction utilities ----
+      //kept for 74X backward-compatibility, not initialized and used in 75X
```

Comment:
in that iorule you would read the 3 old vectors and then fill the new uncertainties_ vector from that.
The old data members can be deleted from the new version of the code.